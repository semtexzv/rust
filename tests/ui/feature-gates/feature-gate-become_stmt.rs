fn other() {
    println!("Called recursively");
}

fn main() {
    become other(); //~ ERROR `become` statement is experimental
}
