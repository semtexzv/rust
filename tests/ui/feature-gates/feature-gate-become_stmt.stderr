error[E0658]: `become` statement is experimental
  --> $DIR/feature-gate-become_stmt.rs:6:5
   |
LL |     become other();
   |     ^^^^^^^^^^^^^^
   |
   = note: see issue #66666 <https://github.com/rust-lang/rust/issues/66666> for more information
   = help: add `#![feature(become_stmt)]` to the crate attributes to enable

error: aborting due to previous error

For more information about this error, try `rustc --explain E0658`.
